app/javascript/react/screens/App/Overview/screens/PlanWizard/components/PlanWizardInstancePropertiesStep/components/TableInlineEditRow/TableConfirmButtonsRow.js

```diff
@@ -96,19 +36,21 @@ class TableConfirmButtonsRow extends React.Component {
     const elements = [
       <tr ref={this.saveRowDimensions} className={rowClass} key="tableRow">
         {this.props.children}
+        <td
+          style={{
+            borderCollapse: 'collapse',
+            borderSpacing: '0px',
+            maxWidth: '0.01px',
```

Comment:
Sub-pixel decimal width values is something I've never seen before 😂  I do wonder whether we can avoid that.